packages/utilities/moment-timezone-include/.gitIgnore

```diff
@@ -0,0 +1 @@
+tz
```

Comment:
😮 today i learned that .gitignore was case insensitive (but it would still be nice to adopt the conventional `.gitignore`)

Reply:
Ooh yeah typo, good spot